rootfs/defaults/jmr.conf

```
@@ -0,0 +1,10 @@
+{{ $JMR_DIRECTORY := .Env.JMR_DIRECTORY | default "/data" -}}
+
+jitsi-multitrack-recorder {
+  recording {
+    {{ if .Env.JMR_FORMAT -}}
+    format = {{ .Env.JMR_FORMAT }}
+    {{ end -}}
```

```suggestion
    {{- end }}
```
this way whitespace after this block is preserved, and we only chomp backwards to the end of the previous line